tests/unittests/Foundation/NSSetTests.mm

```diff
@@ -195,27 +195,12 @@
 TEST(NSSet, ObjectsWithOptionsPassingTest) {
     NSSet* set = [NSSet setWithObjects:@1, @2, @3, @4, @5, nil];
 
-    // Verify that the NSEnumerationConcurrent option executes the blocks concurrently
-    // The blocks will increment waitingCount and wait for the signal unless it is the last element,
-    // in which case it will signal the other blocks to continue and they all decrement waitingCount
-    // so all blocks should be run concurrently, otherwise waitingCount will not return to 0
-    __block NSCondition* condition = [[NSCondition new] autorelease];
-    __block unsigned waitingCount = 0;
+    // Reference platform is not guaranteed to respect NSEnumerationConcurrent, so tests will sometimes fail
```

Comment:
I do not understand what this means. Wouldn't disabling the test on OS X ensure that it is not forced when undefined?